client/src/legacy/ToastsContainer.js

```diff
@@ -0,0 +1,16 @@
+/* global window */
```

Comment:
This bit detects a `<div class="toasts-container"></div>` tag and wired it up. There might be better way of doing this.

Maybe we should just instantiate the `<ToastsContainer />` direction in some higher `<App />` component.

Reply:
Done.